osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2020 Smart Society Services B.V.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.opensmartgridplatform</groupId>
+  <artifactId>osgp-protocol-simulator-mqtt</artifactId>
+  <name>osgp-protocol-simulator-mqtt</name>
```

Comment:
If there is time, it would be great if this project could be built as war instead of jar. There are other simulator projects which are Spring Boot applications built as war, see osgp-protocol-simulator-iec61850 as an example.

Reply:
The idea is to run the simulator with Docker (see the README) since that's what we're migrating to. This was also requested by the client (Egon Kastelijn).

It's possible to also generate a .war file from the build, but is that really the desired course?

Reply:
Discussed with @rtusveld and will change the build so it generates a .war file

Reply:
Done